core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingDataService.js

```diff
@@ -110,19 +93,10 @@ oppia.factory('TrainingDataService', [
 
     return {
       initializeTrainingData: function(explorationId, stateName) {
-        var trainingDataUrl = '/createhandler/training_data/' + explorationId +
-          '/' + encodeURIComponent(stateName);
-        $http.get(trainingDataUrl).then(function(response) {
-          var unhandledAnswers = response.data.unhandled_answers;
-          _trainingDataAnswers = [];
-          _trainingDataFrequencies = [];
-          for (var i = 0; i < unhandledAnswers.length; i++) {
-            var unhandledAnswer = unhandledAnswers[i];
-            _trainingDataAnswers.push(unhandledAnswer.answer);
-            _trainingDataFrequencies.push(unhandledAnswer.frequency);
-          }
-          $rootScope.$broadcast('updatedTrainingData');
-        });
+        var answerGroups = ResponsesService.getAnswerGroups();
```

Comment:
Weird. I guess the previous code was wrong? It seems to have conflated training data with unhandled answers.

Though maybe I'm confused. Is this a service for training new unhandled answers or is it a service for accessing the already-trained data?

Reply:
It is for training new unhandled answers. In old interface we don't have any mechanism to remove an answer from training data. All we could do was submit the answer again and change its answer group.

Reply:
But then this method seems to have changed its aim, from handling unhandled answers to handling the already-trained data?

Reply:
No, the training data is the current training data. The service doesn't really use those answers anywhere (I don't know why they initialized it when it was unnecessary, my guess is that they wanted to handle already trained answers but didn't finish it). Check other methods, they take in an answer in the function call and add it to existing training data.

Reply:
Ah ok, thanks. Sounds fine then, just want to make sure it's fully consistent throughout (and it's clear what it's for). Let's also update the `@fileoverview` at the top of the file to clarify.

Reply:
Updated `@fileoverview`.